packages/wallet/src/redux/state.ts

```diff
@@ -1,10 +1,85 @@
-import { SharedWalletState, emptyState } from './shared/state';
-import { InitializingState } from './initializing/state';
-import { InitializedState } from './initialized/state';
+import { OutboxState, EMPTY_OUTBOX_STATE } from './outbox/state';
+import { FundingState, EMPTY_FUNDING_STATE } from './fundingState/state';
+import { ChannelState } from './channelState/state';
+import { Properties } from './utils';
 
-export * from './initialized/state';
-export * from './initializing/state';
+export type WalletState = WaitForLogin | WaitForAdjudicator | MetaMaskError | Initialized;
 
-export { SharedWalletState, emptyState };
+// -----------
+// State types
+// -----------
+export const WAIT_FOR_LOGIN = 'INITIALIZING.WAIT_FOR_LOGIN';
+export const METAMASK_ERROR = 'INITIALIZING.METAMASK_ERROR';
+export const WAIT_FOR_ADJUDICATOR = 'INITIALIZING.WAIT_FOR_ADJUDICATOR';
+export const WALLET_INITIALIZED = 'WALLET.INITIALIZED';
 
-export type WalletState = InitializingState | InitializedState;
+// ------
+// States
+// ------
```

Comment:
@andrewgordstewart do you think we can safely remove `channelState`, `fundingState` and `outboxState` properties from everything but the `Initialized` state?

Reply:
Yes